src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -376,6 +376,15 @@
         - name: --max-count
           type: int
           short-summary: Maximum nodes count used for autoscaler, when "--enable-cluster-autoscaler" specified. Please specifying the value in the range of [1, 100]
+        - name: --initial-node-taints
+          type: string
+          short-summary: The initial node taints for the node pool. You can't change the node tains through CLI.
```

Comment:
typo, "node tains"

Reply:
Suggest expanding summary: "The node taints applied for nodes added to a node pool. Does not apply to existing nodes."